src/transformers/models/blip/image_processing_blip.py

```diff
@@ -131,11 +132,12 @@ def resize(
         Returns:
             `np.ndarray`: The resized image.
         """
-        size = get_size_dict(size, default_to_square=True)
+        size = get_size_dict(size)
         if "height" not in size or "width" not in size:
             raise ValueError(f"The `size` dictionary must contain the keys `height` and `width`. Got {size.keys()}")
-        output_size = (size["height"], size["width"])
-        return resize(image, size=output_size, resample=resample, data_format=data_format, **kwargs)
+        return resize(
+            image, size=(size["height"], size["width"]), resample=resample, data_format=data_format, **kwargs
```

Comment:
I personally prefer the previous way of defining `output_size` then use it. If you feel the same, we can change `ViTImageProcessor` to use it that way and not to change here.

Reply:
Updated so that ViT uses BLIP's previous implementation with output_size :)